api/awsutil/awsutil.go

```diff
@@ -92,6 +92,7 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		io.WriteString(w, cycle.Response.Body)
 	} else {
 		fmt.Println("Request does not match next cycle.")
+		fmt.Printf("Remaining cycles: %d\n", len(h.cycles))
```

Comment:
The addition of this line made fixing the tests for the aws provider worlds easier.